pkg/io/io.go

```diff
@@ -33,35 +33,30 @@ var FetchFile = func(src string) ([]byte, error) {
 
 func fetchFromRemote(src string) ([]byte, error) {
 	dst := filepath.Join(env.Config().GetString("Hostname"), src)
-	f, err := ReadFile(dst)
-	if err != nil {
-		host := env.Config().GetString("Hostname")
+	host := env.Config().GetString("Hostname")
```

Comment:
Just a note, may be extract `host` before creating `dst`, and then set the `dst` as `dst := filepath.Join(host, src)`? It will be more readable IMHO.

Reply:
Of course! Thanks